src/components/OneSelectedStudentOnList/OneSelectedStudentOnList.css

```diff
@@ -0,0 +1,62 @@
+.student_container{
+    background-color: #292a2b;
+    color: #f7f7f7;
+    width: 1328px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 17px 0 0;
+}
+
+.main_section {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
```

Comment:
Try to name the classes so that they do not repeat themselves anywhere else in the application code. For example the name '.main_section' is a bit too general in my opinion.